ios/EscPosPrinterDiscovery.m

```diff
@@ -55,9 +56,12 @@ - (void)stopDiscovery
     }
 }
 
-- (void) performDiscovery: (void(^)(NSString *))onFinish
+- (void) performDiscovery: (void(^)(NSString *))onFinish params:(NSDictionary *)params
 {
-    dispatch_after(dispatch_time(DISPATCH_TIME_NOW, 5 * NSEC_PER_SEC), dispatch_get_main_queue(), ^{
+    // Default to 5000 if the value is not passed.
+    int scanningTimeout = (int)[params[@"scanningTimeoutIOS"] integerValue] ?: 5000;
+
+    dispatch_after(dispatch_time(DISPATCH_TIME_NOW, (scanningTimeout / 1000) * NSEC_PER_SEC), dispatch_get_main_queue(), ^{
```

Comment:
I think you can directly pass scanningTimeout without dividing and multiplying on 1000. What do you think?

Reply:
Ah I see what you mean, but the value is not 1000
![image](https://user-images.githubusercontent.com/20160573/141997546-fa617066-95ea-4cfb-98fc-e7e733caaea3.png)

But I will change it to `scanningTimeout * NSEC_PER_MSEC`